packages/core/src/components/tag-input/resizableInput.tsx

```diff
@@ -0,0 +1,39 @@
+/* !
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ */
+
+import React, { forwardRef, useEffect, useRef, useState } from "react";
+
+import { Classes, HTMLInputProps } from "../../common";
+
+type ResizableInputProps = HTMLInputProps;
```

Comment:
nit: export for consistency

```suggestion
export type ResizableInputProps = HTMLInputProps;
```